specification/glossary.md

```diff
@@ -86,7 +90,7 @@ Flags represent a single pivot point of logic. Flags have a type, like `string`,
 
 ### Variant
 
-A variant is a space-constrained identifier for values. This allows us to reference these values without having to include all of the data (which may be quite large). We may use some hash of the values themselves or a string name. So for the `header-order` flag, we may have variants like `reverse`, `wonky` or `standard`.
+A variant is a identifier for a value. This allows for the referral to particular values without necessarily to including the value itself, which may be quite prohibitively large or otherwise unsuitable in some cases.
```

Comment:
there's some iffy grammar here

```suggestion
A variant is a identifier for a value. This allows for referral to particular values without necessarily including the value itself, which may be quite prohibitively large or otherwise unsuitable in some cases.
```

Also maybe "A variant is a semantic identifier for a value" might be a bit clearer?

Reply:
I like both these suggestions.